### DISCUSSION TITLE
Where to suggest changes to CodePen examples?

### DISCUSSION THREAD

**Discussion by Metacinnabar** [matteobruni/tsparticles#1926] [Ideas]
Where would I suggest changes/additions to specific CodePen tsparticles example configurations?

**Comment by matteobruni** [comment]
This discussion is fine don’t worry

**Reply by Metacinnabar** [reply]
Ah ok, thanks. I may as well just say them here instead of create a new discussion.
For your Among Us CodePen example, it would be nice if
 - the character position started offscreen, and then moved onto the screen (currently you can see it "spawn" on the left of the screen)
  - the character change to a random color each time it loops

**Reply by matteobruni** [reply]
I didn't find all the colored sprites, so I used the only one found (without the shadow).

About the start position I'm working on a fix about out of canvas since it's used the radius and not the diameter.

**Reply by Metacinnabar** [reply]
Looks like the game uses one sprite and colors them in the code. I found the sprites here:
https://www.spriters-resource.com/pc_computer/amongus/sheet/139386/
Maybe you could write something that changes the color of the character each time it repeats?

For the start position, I'm excited for when you release that

Thank you for you work on tsparticles and for the quick responses!

**Reply by matteobruni** [reply]
I cannot change png colors, since the image is loaded as is. The best solution is to create every color png so it can be random picked, but I'm not that good with graphics software.

**Reply by Metacinnabar** [reply]
I manged to per pixel edit some sprites I found [here](https://cdn.discordapp.com/attachments/871190208085913660/872076439451742218/unknown.png)
These should have no shadow. Let me know otherwise
Your browser will probably show them up as having a white background, but the background is clear.

**Reply by matteobruni** [reply]
Thank you very much, I'll update the sample later after work

**Reply by Metacinnabar** [reply]
Thank you!

**Reply by Metacinnabar** [reply]
Have you managed to find time to update these? If so, where? I don't see changes on the CodePen example at the moment.

**Reply by matteobruni** [reply]
I've updated the sample with the new images, I'm working on `1.34.0` with the spawn fix (and other features)

**Reply by Metacinnabar** [reply]
Thank you!

**Reply by matteobruni** [reply]
https://codepen.io/matteobruni/pen/rNmMVEx Updated with the `1.34.0`, it looks nicer now

**Reply by Metacinnabar** [reply]
Thank you! This looks a lot better.

**Comment by pablodz** [comment]
Add deer to codepen with reactjs pls

**Comment by Griefed** [comment]
Heya matteobruni and everyone stumbling across this question:

Would it be possible to get a preset/example on how to realize the [Matrix](https://images.maennersache.de/matrix,id=5d770f7f,b=maennersache,w=1600,rm=sk.jpeg) screen somehow? If at all possible.

Cheers,
Griefed
